stdlib/fs.gr

```diff
@@ -0,0 +1,1045 @@
+/**
+ * High-level file system interactions. Utilizes WASI Preview 1 for underlying API
```

Comment:
Should this be changed to match the wording of most of the other docs?
```suggestion
 * Utilities for high-level file system interactions. Utilizes WASI Preview 1 for underlying API
```
I noticed most of the time we say `Utilities` for x.